### ISSUE TITLE
Regression for crash when opening any account (v4.0.21)

### ISSUE THREAD

**Issue by djkazic** [Blockstream/green_android#184]
### Description

After adding a Jade, try to open any account, it will crash. Cannot repro on the earlier revision 4.0.20.

### Version

V4.0.21

### Steps to reproduce

See description

### Expected behaviour

No crash 

### Actual behaviour

Crash

### Device or machine

Pixel 6, android 14

### Additional info

```
12-15 13:26:40.614  1582  4573 D CoreBackPreview: Window{53f1155 u0 com.greenaddress.greenbits_android_wallet/com.blockstream.green.ui.MainActivity}: Setting back callback OnBackInvokedCallbackInfo{mCallback=android.window.IOnBackInvokedCallback$Stub$Proxy@1392b03, mPriority=0, mIsAnimationCallback=true}
12-15 13:26:40.636  6392  6392 E AndroidRuntime: Process: com.greenaddress.greenbits_android_wallet, PID: 6392
12-15 13:26:40.636  6392  6392 E AndroidRuntime: Caused by: org.koin.core.error.InstanceCreationException: Could not create instance for '[Factory:'com.blockstream.green.ui.overview.AccountOverviewViewModel']'
12-15 13:26:40.636  6392  6392 E AndroidRuntime: 	at com.blockstream.green.ui.overview.AccountOverviewFragment$special$$inlined$viewModel$default$2.invoke(SourceFile:5)
12-15 13:26:40.636  6392  6392 E AndroidRuntime: 	at com.blockstream.green.ui.overview.AccountOverviewFragment$special$$inlined$viewModel$default$2.invoke(SourceFile:1)
12-15 13:26:40.636  6392  6392 E AndroidRuntime: 	at com.blockstream.green.ui.overview.AccountOverviewFragment.getViewModel(SourceFile:3)
12-15 13:26:40.636  6392  6392 E AndroidRuntime: 	at com.blockstream.green.ui.overview.AccountOverviewFragment.getAccountWalletViewModel(SourceFile:2)
12-15 13:26:40.636  6392  6392 E AndroidRuntime: 	at com.blockstream.green.ui.overview.AccountOverviewFragment.getAccountWalletViewModel(SourceFile:1)
12-15 13:26:40.636  6392  6392 E AndroidRuntime: 	at com.blockstream.green.ui.wallet.AbstractAccountWalletFragment.getWalletViewModel(SourceFile:2)
12-15 13:26:40.636  6392  6392 E AndroidRuntime: 	at com.blockstream.green.ui.wallet.AbstractAccountWalletFragment.getWalletViewModel(SourceFile:1)
12-15 13:26:40.636  6392  6392 E AndroidRuntime: 	at com.blockstream.green.ui.wallet.AbstractWalletFragment.onViewCreated(SourceFile:132)
12-15 13:26:40.636  6392  6392 E AndroidRuntime: 	at com.blockstream.green.ui.overview.AccountOverviewViewModel$_swapInfoStateFlow$1.invoke(SourceFile:2)
12-15 13:26:40.636  6392  6392 E AndroidRuntime: 	at com.blockstream.green.ui.overview.AccountOverviewViewModel$_swapInfoStateFlow$1.invoke(SourceFile:1)
12-15 13:26:40.636  6392  6392 E AndroidRuntime: 	at com.blockstream.green.ui.overview.AccountOverviewViewModel.get_swapInfoStateFlow(SourceFile:10)
12-15 13:26:40.636  6392  6392 E AndroidRuntime: 	at com.blockstream.green.ui.overview.AccountOverviewViewModel.<init>(SourceFile:193)
12-15 13:26:40.636  6392  6392 E AndroidRuntime: 	at org.koin.ksp.generated.AndroidModuleGencom_blockstream_green_diKt$com_blockstream_green_di_AndroidModule$1$13.invoke(SourceFile:3)
12-15 13:26:40.636  6392  6392 E AndroidRuntime: 	at org.koin.ksp.generated.AndroidModuleGencom_blockstream_green_diKt$com_blockstream_green_di_AndroidModule$1$13.invoke(SourceFile:14)
12-15 13:26:40.645  1582  4573 W ActivityTaskManager:   Force finishing activity com.greenaddress.greenbits_android_wallet/com.blockstream.green.ui.MainActivity
12-15 13:26:40.651  1582  1715 I ActivityManager: Showing crash dialog for package com.greenaddress.greenbits_android_wallet u0
12-15 13:26:40.680  1582  1715 D CoreBackPreview: Window{9d5f30c u0 Application Error: com.greenaddress.greenbits_android_wallet}: Setting back callback OnBackInvokedCallbackInfo{mCallback=android.window.WindowOnBackInvokedDispatcher$OnBackInvokedCallbackWrapper@ce7f0c2, mPriority=0, mIsAnimationCallback=false}
```

**Comment by angelix** [comment]
Duplicate of #183, fixed in 75d23e2cc313a249d91ee36d1678e80dde5448d9.
Soon to be released in Google Play Store.
Thanks for reporting.

**Comment by djkazic** [comment]
Closing due to dupe.
